### PR TITLE
Add recipe for Ogg

### PR DESCRIPTION
**Pull request by dschreij**

### CODE REVIEW

File: recipes/ogg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - vc 14  # [win and py35]
+    - pkgconfig  # [unix]
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+about:
+  home: https://www.xiph.org/
+  license: Public Domain
```
Comment (jjhelmus) [inline recipes/ogg/meta.yaml]: The license included in the tar ball (COPYING) seems to be BSD 3-clause.  The Vorbis FAQ seems to indicate that the format is public domain but the libraries are BSD licensed, http://www.vorbis.com/faq/#slic

File: recipes/ogg/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./configure --prefix=${PREFIX}
```
Comment (jjhelmus) [inline recipes/ogg/build.sh]: Would

```
make
make check
``` 
make sense here?

**Comment by jjhelmus** [comment]
We like to include some type of tests in conda recipes if possible.  For libraries it is often best in run the built in tests using `make check` if the package supports them.  Additionally at minimum checking for the existence of the library files and/or the linkages to the shared libraries can be prudent.  As examples look at the tests run by the [zlib](https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L25-L28) or [readline](https://github.com/conda-forge/readline-feedstock/blob/master/recipe/meta.yaml#L32-L42) packages.

File: recipes/ogg/meta.yaml (hunk 1 above)
Comment (dschreij) [inline recipes/ogg/meta.yaml]: Right, they say BSD-like license on the website, but I'll just keep it to BSD 3-clause then.

File: recipes/ogg/build.sh (hunk 2 above)
Comment (dschreij) [inline recipes/ogg/build.sh]: Hmm what would be the added benefit? Does that test the built library?

**Comment by dschreij** [comment]
@conda-forge/core I don't understand how the above conflicts came to be. I didn't see them when working with git. The mad recipe has been moved to its own branch (along with other recipes) a while ago. Any idea how this could have gone wrong?

**Comment by scopatz** [comment]
@dschreij - I don't either. But the history of this repo doesn't matter so much.  Maybe try to make a new branch and copy over the recipe files from this one?

**Comment by dschreij** [comment]
Yeah, I think I'll just start over for this and all other recipes I have
currently lined up, because of most the commit history has become a mess.
Thanks for the suggestions.

**Pull request closed without merging**